src/webui/www/public/scripts/mocha-init.js

```diff
@@ -35,6 +35,20 @@ var pauseFN = function() {};
 
 initializeWindows = function() {
 
+    saveWindowSize = function(windowId) {
+        var size = $(windowId).getSize();
+        localStorage.setItem('window_' + windowId + '_width', size.x);
+        localStorage.setItem('window_' + windowId + '_height', size.y);
+    };
+
+    getWindowWidth = function(windowId, defaultValue) {
```

Comment:
loadWindowWidth & loadWindowHeight ? since the counterpart is saveXXXX

Reply:
Function names updated